clients/ctl/admin-ui/src/pages/dataset/index.tsx

```diff
@@ -20,17 +20,8 @@ import {
 } from "~/features/dataset/dataset.slice";
 import DatasetsTable from "~/features/dataset/DatasetTable";
 
-const useDatasetsTable = () => {
-  const { data, isLoading } = useGetAllDatasetsQuery();
-
-  return {
-    isLoading,
-    datasets: data,
-  };
-};
-
 const DataSets: NextPage = () => {
-  const { isLoading, datasets } = useDatasetsTable();
+  const { isLoading } = useGetAllDatasetsQuery();
```

Comment:
how come this is being called twice now (once here, and once within DatasetTable)?

Reply:
The loading spinner is still at this level. It could be moved into the table. The two hook calls use the same query state under the hood, so it doesn't have too much effect.